src/EnergyPlus/ChillerElectricEIR.cc

```diff
@@ -655,6 +655,57 @@ void GetElectricEIRChillerInput(EnergyPlusData &state)
             }
         }
 
+        if (NumAlphas > 16) {
+            thisChiller.CondenserFlowControl = static_cast<DataPlant::CondenserFlowControl>(
+                getEnumValue(DataPlant::CondenserFlowControlNamesUC, state.dataIPShortCut->cAlphaArgs(17)));
+        } else {
+            thisChiller.CondenserFlowControl = DataPlant::CondenserFlowControl::ConstantFlow;
+        }
```

Comment:
FWIW, I think the `else` could be handled by just initializing the flow control variable to the default value on the chiller class.  You don't have to change it, but it could be nice to default initialize the variable on the class rather than rely on the input routine to set the default.